Products/ZenModel/migrate/addOOMParams.py

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2019, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import servicemigration as sm
+
+from Products.ZenModel.ZMigrateVersion import (
+    SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+)
```

Comment:
Unfortunately, the `ZMigrateVersion` imports need to be on one line.  When prodbin is released, this import is removed and the removal code doesn't handle multiline import statements.